src/sonic_ax_impl/mibs/ietf/rfc4363.py

```diff
@@ -48,8 +49,11 @@ def reinit_data(self):
             # Example output: ASIC_STATE:SAI_OBJECT_TYPE_BRIDGE_PORT:oid:0x3a000000000616
             bridge_port_id = s[45:]
             ent = self.db_conn.get_all(mibs.ASIC_DB, s, blocking=True)
-            port_id = ent[b"SAI_BRIDGE_PORT_ATTR_PORT_ID"][6:]
-            self.if_bpid_map[bridge_port_id] = port_id
+            try:
+                port_id = ent[b"SAI_BRIDGE_PORT_ATTR_PORT_ID"][6:]
+                self.if_bpid_map[bridge_port_id] = port_id
+            except KeyError:
+                logger.warning("SAI_OBJECT_TYPE_BRIDGE_PORT {} does not have SAI_BRIDGE_PORT_ATTR_PORT_ID attribute".format(bridge_port_id))
```

Comment:
>logger.warning [](start = 16, length = 14)

It will log the warning every min. Is it too much?

Reply:
this is abnormal, we need to debug on this situation.